components/multystyleNotice.brs

```diff
@@ -1,45 +1,137 @@
 sub init()
+    m.PAGE_HEIGHT = 536
```

Comment:
See `updatePagesNumber` function. XML property cannot be used since it is modified when the screen is initialized, in order to calculate what's the number of pages.